tests/unit/nupic/encoders/multi_test.py

```diff
@@ -101,8 +101,8 @@ def testReadWrite(self):
                                       periodic=True, name="day of week",
                                       forced=True))
     original.addEncoder("myval",
-                        ScalarEncoder(w=5, resolution=1, minval=1, maxval=10,
-                                      periodic=False, name="aux", forced=True))
+                        AdaptiveScalarEncoder(n=50, w=5, resolution=1, minval=1, maxval=10,
```

Comment:
why use AdaptiveS when its use is discouraged [1], rather stick with Scalar/new RDSE? 

[1] https://github.com/numenta/nupic/wiki/Encoders#adaptive-scalar-encoder